translations/en-us.yml

```diff
@@ -1322,6 +1322,8 @@ registries:
         none_selected: 'None selected'
     overview:
         files:
+            outputFeaturePopoverHeading: 'New feature!'
+            outputFeaturePopoverBody: 'You can now add DOIs for external outputs (ex. data, code, papers, etc.) to your registration. Simply click Files and then select the new Outputs tab.'
```

Comment:
Leaving a comment here so we remember to update this usage of `outputs` in case that language changes